modules/setting/repository.go

```diff
@@ -227,8 +228,10 @@ var (
 		// Issue settings
 		Issue: struct {
 			LockReasons []string
+			MaxPinned   int
 		}{
 			LockReasons: strings.Split("Too heated,Off-topic,Spam,Resolved", ","),
+			MaxPinned:   6,
```

Comment:
```suggestion
			MaxPinned:   3,
```

As discussed, I think we better match this default to GitHub. No strong opinion, but two rows already seems too much to me.